src/Web/Sfa.Das.Sas.Web/Views/Provider/ProviderDetail.cshtml

```diff
@@ -27,8 +27,55 @@
                     <dd id="trading-names">@Model.TradingNames</dd>
                 }
                 <dt>UKPRN:</dt>
-                <dd>@Model.UkPrn</dd>
+                <dd>@Model.UkPrn</dd>                   
             </dl>
+            <p class="markdown-header-text">About this Provider</p>
+            @if (string.IsNullOrEmpty(Model.MarketingInfo))
+            {
+                <span>There is no marketing information available about this provider.</span>
```

Comment:
Do we need this info? or can we wrap markdown-header in if not nullOrEmpty?

Reply:
I agreed this with Lisa as an approach to empty details.  I'll update the story accordingly.